10up/simple-page-ordering
# Add functionality to reset page order

### Description of the Change
In the page admin section, the Simple Page Ordering plugin help section has been modified to include a "Reset page order" link. Clicking this link first displays a confirmation dialog then, if the user confirms, resets the "menu_order" field of all pages to 0, resetting any changes made with this plugin.

I think this is the most minimally invasive solution, adding little code to achieve the purpose. I chose to place the reset link in the help menu to minimize chances of accidental order resets. Plus, if someone needs to reset the order of their posts, they have probably made a mistake and will be looking for help.

Closes #24

### How to test the Change
* Install and enable the Simple Page Ordering plugin
* Navigate to the page admin section `/wp-admin/edit.php?post_type=page`
* Add some pages
* Drag some pages up or down so the order changes
* Refresh the page to ensure the order remains (AJAX calls have been successful)
* At the top of the page, click "Help", then "Simple Page Ordering", then "Reset page order"
* Click "OK"
* If the page refreshes and the pages now appear in their original order, this change is functioning properly

### Changelog Entry
> Added - Feature to reset page order


### Credits
Props @pattonwebz, @ruscoe, @Sidsector9, @dkotter


### Checklist:
